e2e/tests/functional/plugins/telemetryTable/telemetryTable.e2e.spec.js

```diff
@@ -45,6 +45,86 @@ test.describe('Telemetry Table', () => {
     await expect(rows).toHaveCount(50);
   });
 
+  test('on load, auto scrolls to top for descending, and to bottom for ascending', async ({
+    page
+  }) => {
+    async function getScrollPosition(top = true) {
+      const tableBody = page.locator('.c-table__body-w');
+
+      // Wait for the scrollbar to appear
+      await tableBody.evaluate((node) => {
+        return new Promise((resolve) => {
+          function checkScroll() {
+            if (node.scrollHeight > node.clientHeight) {
+              resolve();
+            } else {
+              setTimeout(checkScroll, 100);
+            }
+          }
+          checkScroll();
+        });
+      });
+
+      // make sure there are rows
+      const rows = page.getByLabel('table content').getByLabel('Table Row');
+      await rows.first().waitFor();
+
+      // wait a bit to give time for new rows to be added
+      await page.waitForTimeout(1000);
```

Comment:
you can use an expect here actually. something like

`expect(page.getByLabel('Table Row').first()).toBeVisible()`

This will wait up to the default timeout (5 seconds) for a table row to appear. No need for hard waits